src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -33,8 +33,41 @@ $(document).ready(function () {
     updateConstSumMessages();
 
     prepareRubricQuestions();
+    
+    prepareMCQQuestions();
 });
 
+// Prepare mcq questions for answering by user
+function prepareMCQQuestions() {
+	// Get index of mcq question
+	var mcqQuestionNums = getQuestionTypeNumbers("MCQ");
+
+	for (var i = 0; i < mcqQuestionNums.length; i++) {
+		var qnNum = mcqQuestionNums[i];
+
+		// Get number of options for certain mcq question
+		var optionNums = $("[name^='responsetext-" + qnNum + "']").length;
+		for (var k = 0; k < optionNums; k++) {
+			// Get specific option for that question
+			var radioInput = $("[name^='responsetext-" + qnNum + "-" + k + "']");
+			radioInput.prop("checked", false);
+
+			// Toggle radio button if it was checked
+			radioInput.click(function(event) {
+				if ($(this).data("waschecked")) {
+					$(this).prop('checked', false);
+					$(this).data('waschecked', false);
```

Comment:
Yep I also looked into the code. 
It overrides the click event for whole 'td', and changes its child radio input's property to "!radioInput.prop('checked')". Even if clicking on the radio input, the target.event is still whole 'td', while in my case clicking on the radio input, the target.event will only be the radio input, as a result the 'checked' property is checked before overriding the click event of radio input.
I'm not sure how to group whole 'td' as the target.event no matter which children is clicked.